test/integration/auth.spec.ts

```diff
@@ -336,6 +336,12 @@ describe('admin.auth', () => {
           .then((userRecord) => {
             expect(userRecord.metadata.lastRefreshTime).to.exist;
             expect(isUTCString(userRecord.metadata.lastRefreshTime!));
+            expect(new Date(userRecord.metadata.creationTime).getTime())
```

Comment:
Can we also add a unit test for this so it gets regularly checked in CI?

Reply:
Done. Some followup on it may be required during the toJSON PR. (Aside: Does CI not also run the integration tests?)